components/dashboard/RequestView.tsx

```diff
@@ -0,0 +1,304 @@
+import { useMemo, useState, ReactElement, useEffect } from "react";
+import { ButtonPrimary, ButtonSecondary } from "@/components/core/Button";
+import { DashboardCard } from "@/components/core/Layout";
+import { TabPrimary } from "@/components/core/Tabs";
+import NextLink from "next/link";
+import DashboardSearchInput from "@/components/core/DashboardSearchInput";
+import {
+  Link,
+  Tabs,
+  ButtonGroup,
+  TabList,
+  TabPanels,
+  TabPanel,
+  Table,
+  Thead,
+  Spacer,
+  Box,
+  Tbody,
+  Tr,
+  Th,
+  Td,
+  IconButton,
+  useDisclosure,
+  useToast,
+  ModalOverlay,
+  ModalContent,
+  ModalHeader,
+  ModalCloseButton,
+  ModalBody,
+  ModalFooter,
+  Modal,
+} from "@chakra-ui/react";
+import { PropertyImage } from "@/components/sections/Listings";
+import { TenantRequestEntry } from "@/src/api/types";
+import { auth } from "@/src/firebase";
+import { useAuthState } from "react-firebase-hooks/auth";
+import { tenantRequests } from "@/src/api/collections";
+import { useCollection } from "react-firebase-hooks/firestore";
+import { getUserPublicById } from "@/src/api/queries";
+import { DeleteIcon, CloseIcon, CheckIcon } from "@chakra-ui/icons";
+import {
+  listingRouteBuilder,
+  listingHrefBuilder,
+} from "@/src/utils/listingRoute";
+import { Heading5 } from "../core/Text";
+
+enum Request {
+  Sent = "tenantUid",
+  Received = "landlordUid",
+}
+
+interface DeleteProps {
+  isOpen: boolean;
+  onClose: () => void;
+  request: TenantRequestEntry | undefined;
+}
+
+function RequestView(): ReactElement {
+  // @ts-ignore
```

Comment:
Nitpick: `@ts-ignore` isn't necessary here

Looks like that was leftover from TenantView.tsx where it used that to ignore the unused `selectedPeriod` variable

Reply:
fixed 7c9614d